README.md

```
@@ -31,7 +31,7 @@ Install with setup.py:
 ```bash
 $ git clone https://github.com/PRIDE-Archive/pridepy
 $ cd pridepy
-$ python setup.py sdist bdist_wheel 
+$ python pyproject.toml sdist bdist_wheel 
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)